perception/multi_object_tracker/include/multi_object_tracker/tracker/model/normal_vehicle_tracker.hpp

```diff
@@ -39,6 +39,8 @@ class NormalVehicleTracker : public Tracker
     VX = 3,
     WZ = 4,
   };
+  int nearest_corner_index_ = -1;
```

Comment:
memo
- don't recommend to use magic number
- better to use enum class(right_front, left_front, right_rear, left_rear)